msteams-ui-styles-core/src/components/input.ts

```diff
@@ -1,7 +1,15 @@
-import { keyframes } from 'typestyle';
 import { chooseStyle, IContext } from '../context';
 import { errorLabel } from '../index';
 import { label } from './label';
+import { setStylesTarget , keyframes } from 'typestyle';
+
+const target = document.createElement('style');
+target.setAttribute('nonce', 'cmFuZG9tIG51bWJlcg=='); //This is a base64 encoded random number which can be added in the csp header as nonce value
```

Comment:
should this be generated randomly or not? I'm confused

Reply:
Yes. It should be generated randomly. 
However the base idea is to have a value which is same as the one present in style-src directive of the CSP header.
When both the values match, CSP allows this style created/injected as an inine style.